### PR TITLE
Add link to FAQ to issue template chooser

### PR DESCRIPTION
**Pull request by DNin01**
Configures the [issue template chooser](https://github.com/ScratchAddons/contributors/issues/new/choose) to add a link to https://scratchaddons.com/docs/faq/#what-can-i-do-if-i-find-a-problem, which highlights the recommended options for leaving feedback about Scratch Addons.

There have been a few instances where people open issues here instead of on the ScratchAddons repository, where their feedback should go, perhaps because they were led here by the [credits page](https://scratchaddons.com/credits/).

Tested on a different repository.

### CODE REVIEW

**Review by Hans5958 — CHANGES_REQUESTED** [review]
I would like to reword it to be general, something like "want to say something about Scratch?"

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Send Feedback
+    url: https://scratchaddons.com/docs/faq/#what-can-i-do-if-i-find-a-problem
+    about: Trying to leave feedback? Go here to learn more.
```
Comment (DNin01) [inline .github/ISSUE_TEMPLATE/config.yml:5]: @Hans5958 Something like this?
```suggestion
    about: Want to say something about Scratch Addons? Learn more about leaving feedback here.
```

Comment (Hans5958) [reply .github/ISSUE_TEMPLATE/config.yml:5]: I was thinking to reword the mentions of "Send Feedback" to something like "Contact us" or something more general.

Comment (DNin01) [reply .github/ISSUE_TEMPLATE/config.yml:5]: > I was thinking to reword the mentions of "Send Feedback" to something like "Contact us" or something more general.

Would you mind writing a potential rewording?

Personally, "contact us" isn't the only thing I'd say because we have public areas where you can leave feedback.

Comment (Hans5958) [reply .github/ISSUE_TEMPLATE/config.yml:5]: Something like "Want to say something about Scratch Addons? Tell us here!" or "Want to say something about Scratch Addons? Learn more about how to reach/tell/contact us!"

Comment (DNin01) [reply .github/ISSUE_TEMPLATE/config.yml:5]: > "Want to say something about Scratch Addons? Learn more about how to reach/tell/contact us!"

I'll commit that.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+  - name: Send Feedback
```
Comment (Hans5958) [inline .github/ISSUE_TEMPLATE/config.yml:3]: Oh, and don't forget this.

```suggestion
  - name: Contact Us
```

...or, in a more cheeky way...


```suggestion
  - name: Have something to say?
```

**Review by Hans5958 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/config.yml (hunk 2 above)
Comment (Hans5958) [reply .github/ISSUE_TEMPLATE/config.yml:3]: You can pick one you prefer and I will merge this, or I could do this tommorow.

**Comment by DNin01** [comment]
Personally, I chose "Send Feedback" because not all methods of leaving feedback talk directly to the organization members and the form where you contact us is referred to as the "send feedback" form, and same with the button to go to the feedback form from Scratch Addons. It can also imply community involvement.

However, a "Contact Us" wording is more general and inclusive of the different kinds of things people want to tell us (like bugs), and we still _see_ the feedback anyway.

Any other reasons you like to say "Contact Us"?

**Comment by Hans5958** [comment]
I don't think it matters much whether if it is direct or not[^1]. I think leaving a feedback is enough to be counted as them contacting us, whether if they got a response or not.

I think it's safe (enough) to assume that those who came here wants to reach us via GitHub and not the others. Now, linking to GitHub only would be suffice, but I guess giving people options is a better idea. 

And, with that there's a possibility that two-way channel to communicate is used, and I don't think it is simply as "leaving a feedback",

**Comment by DNin01** [comment]
Makes sense.

**Review by Hans5958 — APPROVED** [review]
